i18n/ar.json

```diff
@@ -219,5 +219,7 @@
     "Yes, see it": "نعم، حمله",
     "Dark Mode": "الوضع الداكن",
     "Error compressing paste, due to missing WebAssembly support.": "Error compressing paste, due to missing WebAssembly support.",
-    "Error decompressing paste, your browser does not support WebAssembly. Please use another browser to view this paste.": "Error decompressing paste, your browser does not support WebAssembly. Please use another browser to view this paste."
+    "Error decompressing paste, your browser does not support WebAssembly. Please use another browser to view this paste.": "Error decompressing paste, your browser does not support WebAssembly. Please use another browser to view this paste.",
+    "Paste copied to clipboard": "Paste copied to clipboard",
+    "To copy paste press on the copy button or use the clipboard shortcut Ctrl+c/Cmd+c": "To copy paste press on the copy button or use the clipboard shortcut Ctrl+c/Cmd+c"
```

Comment:
Over in #1463 I propose to make messages referring to keyboard-shortcuts more accessible by wrapping the keys into kbd-tags. Could we apply the same here, too? Example:
https://github.com/PrivateBin/PrivateBin/blob/5aa7a95841779633e6cb80aea343991144dfc4e9/i18n/ar.json#L154